docs/GETTING_STARTED.md

```diff
@@ -379,6 +379,85 @@ $ bin/step ca provisioner remove jim@smallstep.com --all
 The same entity may have multiple provisioners for authorizing different
 types of certs. Each of these provisioners must have unique keys.
 
+## Use Custom Claims for Provisioners to Control Certificate Validity etc
+
+It's possible to configure provisioners on the CA to issue certs using propoerties specific to their target environments. Most commonly different validity periods and disabling renewals for certs. Here's how:
+
+```bash
+$ step ca init
+# complete the init steps
+$ step ca provisioner add --create dev@smallstep.com
+# lets create a provisioner for dev certs
+Please enter a password to encrypt the provisioner private key? password
+# add claims inside a provisioner element in ~/.step/config/ca.json
+~/.step/config/ca.json
+[...]
+"authority": {
+   "provisioners": [
+      {
+         "name": "you@smallstep.com",
+         "type": "jwk",
+         "key": {
+            "use": "sig",
+            "kty": "EC",
+            "kid": "Kg43gSukHnl8f5NztLPDxqpz_9TNUILnMrIMIa70jOU",
+            "crv": "P-256",
+            "alg": "ES256",
+            "x": "So0JVWFFXo-6GmDwq6WWZZk-AFZt5GKTx5PzdLhdsrQ",
+            "y": "kVz8pCl2Qx9fZmJZhXGrHpufwNDTp7oHwi8Zaj7rhiQ"
+         },
++        "claims": {
++           "minTLSCertDuration": "5s",
++           "maxTLSCertDuration": "12h",
++           "defaultTLSCertDuration": "2h",
++           "disableRenewal": true
++        },
+         "encryptedKey": "..."
+      }
+   ]
+},
+[...]
+
+A duration string is a possibly signed sequence of
```

Comment:
O or negative numbers do not make sense and will cause an error, so, we probably can remove the `signed`word.